src/src/rx_main.cpp

```diff
@@ -108,6 +108,8 @@ MSP2CROSSFIRE msp2crsf;
 #endif
 
 #if defined(PLATFORM_ESP8266) || defined(PLATFORM_ESP32)
+#include "rx-serial/SerialHoTT_TLM.h"
```

Comment:
This should be up by the other rx-serial includes

Reply:
done